remoting/getty/getty_client.go

```diff
@@ -102,8 +102,7 @@ func SetClientConf(c ClientConfig) {
 
 func setClientGrpool() {
 	if clientConf.GrPoolSize > 1 {
-		clientGrpool = gxsync.NewTaskPool(gxsync.WithTaskPoolTaskPoolSize(clientConf.GrPoolSize), gxsync.WithTaskPoolTaskQueueLength(clientConf.QueueLen),
-			gxsync.WithTaskPoolTaskQueueNumber(clientConf.QueueNumber))
+		clientGrpool = gxsync.NewTaskPoolSimple(clientConf.GrPoolSize)
```

Comment:
pls  validate  clientConf.GrPoolSize and set a default value if it's illegal.

Reply:
@watermelo reply this comment pls